### DISCUSSION TITLE
Expand more tags

### DISCUSSION THREAD

**Discussion by parfait8566** [Jitendex/Jitendex#11] [Ideas]
Compared to JMdict Extra, Jitendex often expands JMdict "tags" to be more immediately clear, like `uk` -> `kana`.
I think it'd be very helpful if this was applied to more tags across all categories (for example `vz` for POS attributes, `chn` for misc, `psych` for fields, `ksb` for dialects and so on).

**Comment by stephenmk** [comment]
Yes, definitely. This has just been in the back of my mind and I haven't got around to doing it.

**Reply by stephenmk** [reply]
Implemented in the latest version
